runtime/commonTest/src/kotlinx/serialization/SerialDescriptorSpecificationTest.kt

```diff
@@ -24,23 +25,20 @@ class SerialDescriptorSpecificationTest {
         testHolderDescriptor(HolderDescriptor)
     }
 
-    private object HolderDescriptor : SerialClassDescImpl(
-        "kotlinx.serialization.SerialDescriptorSpecificationTest.Holder"
-    ) {
-        init {
-            addElement("a")
-            pushDescriptor(IntSerializer.nullable.descriptor)
+    private object StaticHolder {
```

Comment:
why do we need `StaticHolder`?

Reply:
to have a `val` to be referenced from another object via delegation

Reply:
This test was written before this function :)

Reply:
Maybe `companion object`?